### PR TITLE
[CI] partial release and install fixes

### PR DESCRIPTION
**Pull request by leecalcote**
Signed-off-by: Lee Calcote <lee.calcote@layer5.io>

**Notes for Reviewers**

This PR partially addresses the need for an `install` and `update` script (could be the same script). I'm guessing that `helm` takes care of `uninstall` insomuch as is needed for this plugin, which would be to delete the cloned repo (this repo) from user's filesystem.

Please feel free to checkout this branch and add commits or to close this PR and start your own.

**Resources:**

- Helm Plugin dev guide - https://helm.sh/docs/topics/plugins/
- Example Helm plugins and their installers
  - [sigstore](https://github.com/sigstore/helm-sigstore/blob/main/plugin.yaml) ( [install script](https://github.com/sigstore/helm-sigstore/blob/main/scripts/install-binary.sh) )
  - [helm dashboard](https://github.com/komodorio/helm-dashboard/blob/main/plugin.yaml) ([install script](https://github.com/komodorio/helm-dashboard/blob/main/scripts/install_plugin.sh))

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: scripts/install-binary.sh [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+HELM_BIN="/usr/local/bin/helm"
+export GREP_COLOR="never"
+
+HELM_MAJOR_VERSION=$("${HELM_BIN}" version --client --short | awk -F '.' '{print $1}')
+
+# : ${HELM_PLUGIN_DIR:="$("${HELM_BIN}" home --debug=false)/plugins/helm-diff"}
+
+# Handle HELM_PLUGIN_DIR filepath based on OS. Use *nix-based filepathing
+
+if type cygpath >/dev/null 2>&1; then
```
Comment (hexxdump) [inline scripts/install-binary.sh:34]: For platforms other than windows, the HELM_PLUGIN_DIR is going to be "" and eventually the CWD is used for downloading the files, right?

File: scripts/install-binary.sh [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+  version="$(cat $HELM_PLUGIN_DIR/plugin.yaml | grep "version" | cut -d '"' -f 2)"
+  if [ -n "$version" ]; then
+DOWNLOAD_URL="https://github.com/$PROJECT_GH/releases/download/v$version/$CLI_$version_$OS_$ARCH.tar.gz"
+    echo "DOWNLOAD_URL1: $DOWNLOAD_URL"
+    # https://github.com/meshery/helm-kanvas-snapshot/releases/download/v0.2.0/kanvas-snapshot_0.2.0_Darwin_x86_64.tar.gz
+  else
+    # Use the GitHub API to find the download url for this project.
+    url="https://api.github.com/repos/$PROJECT_GH/releases/latest"
+    if type "curl" >/dev/null; then
```
Comment (hexxdump) [inline scripts/install-binary.sh:101]: verfiySupported checks for curl and wget already, do we need these checks again ?

Comment (leecalcote) [reply scripts/install-binary.sh:101]: Probably not. Please change at-will

**Comment by leecalcote** [comment]
Addressed in #51

**Pull request closed without merging**